From 090f1a4baf02eb4a7c7fee34d126493d2d4020cf Mon Sep 17 00:00:00 2001
From: J. Bruce Fields <bfields@redhat.com>
Date: Mon, 23 Jun 2014 21:49:55 -0400
Subject: [fs] nfsd: typo in nfsd_rename comment

Message-id: <1403560314-32525-54-git-send-email-bfields@redhat.com>
Patchwork-id: 84469
O-Subject: [Patch RHEL7 053/172] nfsd: typo in nfsd_rename comment
Bugzilla: 1111719
RH-Acked-by: Steve Dickson <SteveD@redhat.com>

Signed-off-by: J. Bruce Fields <bfields@redhat.com>
Upstream: fbb74a34a5b04dee0602e19f3f46496f8bf1d6b4
Bugzilla: 1111719
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/nfsd/vfs.c b/fs/nfsd/vfs.c
index 70a949b..1c466d3 100644
--- a/fs/nfsd/vfs.c
+++ b/fs/nfsd/vfs.c
@@ -1866,10 +1866,10 @@ nfsd_rename(struct svc_rqst *rqstp, struct svc_fh *ffhp, char *fname, int flen,
  dput(odentry);
  out_nfserr:
  err = nfserrno(host_err);
-
- /* we cannot reply on fh_unlock on the two filehandles,
+ /*
+  * We cannot rely on fh_unlock on the two filehandles,
   * as that would do the wrong thing if the two directories
-  * were the same, so again we do it by hand
+  * were the same, so again we do it by hand.
   */
  fill_post_wcc(ffhp);
  fill_post_wcc(tfhp);
-- 
1.7.1